plugins/catalog-node/package.json

```diff
@@ -46,6 +46,7 @@
     "@backstage/catalog-client": "workspace:^",
     "@backstage/catalog-model": "workspace:^",
     "@backstage/errors": "workspace:^",
+    "@backstage/plugin-catalog-backend": "workspace:^",
```

Comment:
this can't be added as a dependency, as `catalog-node` is already a dependency of `catalog-backend`
```suggestion
```

Reply:
For this change, there is an additional `import { CatalogPermissionRuleInput } from '@backstage/plugin-catalog-backend'` in `plugins/catalog-node/src/extensions.ts` though:
https://github.com/backstage/backstage/pull/21275/files#diff-28320375e7a233054c9751dd27df02429826cc2bafe6829b66c3dcf1f5044999R17

Reply:
Hi @vinzscam, I have duplicated the `CatalogPermissionRuleInput` type in `plugins/catalog-node/src/extensions.ts` to avoid the circular dependency, could you please review the change? Thank you!